notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb

```diff
@@ -0,0 +1,897 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "26dc2193-7a7e-4e52-bf7e-8a62a3d96866",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script.log.info(f\"{test_case} - {test_exec} - Reset corrections\")\n",
+    "await mtcs.rem.mtaos.cmd_resetCorrection.start(timeout=60.)\n",
+    "\n",
+    "script.log.info(f\"{test_case} - {test_exec} - Issue reset corrections\")\n",
+    "await mtcs.rem.mtaos.cmd_issueCorrection.start(timeout=60.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32b34689",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## Add 2um aberration to z7\n",
+    "\n",
+    "Add 2um the z7 coefficient via OFC.  \n",
+    "Compare the corrections sent vs forces and position changes applied.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1bc5625c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The wavefront errors input is an array with 19 elements where the first \n",
+    "#   element corresponds to z4\n",
+    "wavefront_errors = np.zeros(19)\n",
+    "\n",
+    "# Add1 um to z7\n",
+    "wavefront_errors[3] += 2.0 \n",
+    "\n",
+    "script.log.info(f\"{test_case} - {test_exec} - Add 2um aberration\")\n",
+    "await mtcs.rem.mtaos.cmd_addAberration.set_start(wf=wavefront_errors, timeout=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dbd4fd0e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script.log.info(f\"{test_case} - {test_exec} - Issue 2um aberration\")\n",
+    "await mtcs.rem.mtaos.cmd_issueCorrection.start(timeout=60.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99a48e19",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "## Stop Tracking"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca763a73",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.stop_tracking()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45fda728-c298-42d2-93f8-99bc16f87aa8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.reset_m1m3_forces()\n",
+    "await mtcs.reset_m2_forces()\n",
+    "await mtcs.reset_camera_hexapod_position()\n",
+    "await mtcs.reset_m2_hexapod_position()\n",
+    "script.log.info(f\"{test_case} - {test_exec} - STOP\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c884702f",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "Check that the corrections in step 10 are twice of those in step 7. This step does not currently involve running any commands in this notebook. This step must be verified using a separate noteboook. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "951661e0-974c-4dec-8c68-03b7407f01ed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.point_azel(az=0, el=80)\n",
```

Comment:
Same here. Perhaps add here safe = False to move mount in small steps?

Reply:
Same here. This is a stray commit. It should be updated on another branch/PR.